lib/Receiver.js

```diff
@@ -310,18 +312,9 @@ class Receiver {
    * @api private
    */
 
-  error (reason, protocolErrorCode) {
-    if (this.dead) return;
```

Comment:
This was added here https://github.com/websockets/ws/commit/913afa3de0fb9b46aab215c40e177d98988c2098#diff-ee39beab4c8223a58e34e180ba59f345 but the issue it attempted to fix (#680) was actually caused by [reentrance](https://github.com/websockets/ws/blob/913afa3de0fb9b46aab215c40e177d98988c2098/lib/Receiver.js#L462) after [cleanup](https://github.com/websockets/ws/blob/913afa3de0fb9b46aab215c40e177d98988c2098/lib/Receiver.js#L396).
This is no longer the case in master so it has been removed.